cms/djangoapps/models/settings/course_grading.py

```diff
@@ -66,7 +73,7 @@ def update_from_json(course_key, jsondict, user):
         CourseGradingModel.update_grace_period_from_json(course_key, jsondict['grace_period'], user)
 
         CourseGradingModel.update_minimum_grade_credit_from_json(course_key, jsondict['minimum_grade_credit'], user)
-
+        _emit_grading_event(course_key, user.id)
```

Comment:
Since `_emit_grading_event` only ever gets called in tandem with sending the signal, I'm going to update the helper function to do both. We can break them apart if the need arises down the line but for now, they only happen together.